rfcs/0000-c-unwind-abi.md

```diff
@@ -255,13 +210,10 @@ behavior. `"C"`-like ABIs are `"C"` itself but also related ABIs such as
 | `panic=abort`  | `"C unwind"` | `panic!` aborts                       | abort                   |
 | `panic=abort`  | `"C"`-like   | `panic!` aborts (no unwinding occurs) | UB                      |
 
-Regardless of the panic runtime, ABI, or platform, the interaction of Rust
-frames with C functions that deallocate frames (i.e. functions that may use
-forced unwinding on specific platforms) is specified as follows:
-
-* **When deallocating Rust [POFs][POF-definition]:** frames are safely
-  deallocated; no undefined behavior
-* **When deallocating Rust non-POFs:** undefined behavior
+The interaction of Rust frames with C functions that deallocate frames (i.e.
+functions that may use forced unwinding on specific platforms) is not specified
+in this RFC. However, we do specify that deallocating
+non-[POFs][POF-definition]s in this way is undefined behavior.
```

Comment:
I sort of like the old setup, which I found somewhat clearer. Can we just change the "When deallocating Rust POFs" to say "this RFC does not fully specify this behavior. For now, it must be considered undefined, but the project-ffi-unwind group expects to create some path that enables POF frames to be safely and correctly deallocated via longjmp. However, additional annotations may be required."

Reply:
I think alluding to new annotations in this RFC would be premature, but I'll add something along those lines.

Reply:
The main thing is "you may have to change the source in some way"

Reply:
The text you added seems fine.